module/PowerShellEditorServices.VSCode/en-US/Microsoft.PowerShell.EditorServices.VSCode.dll-Help.xml

```diff
@@ -0,0 +1,580 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Usually the xml file is considered a build artifact and not actually checked in

Reply:
deleted